benchmarks/sql-benchmark/kaskada.ipynb

```diff
@@ -58,6 +58,9 @@
    "source": [
     "from kaskada import table\n",
     "\n",
+    "# Change this to the (relative) path to the generated input.\n",
+    "INPUT_PATH = '6x_1'\n",
```

Comment:
This is the equivalent. May be worth modifying the generator script to include an "output prefix" or something to more easily work with these "input paths".